tox.ini

```diff
@@ -38,6 +38,6 @@ deps =
     umap-learn
     napari-tools-menu
     napari-skimage-regionprops>=0.2.0
-    hdbscan
+    hdbscan==0.8.29
```

Comment:
Hard pinning is tricky. Do you know what's wrong with hdbscan in higher versions?

Reply:
I am not sure, I have a working environment with hdbscan 0.8.29 where `pytest` is not failing.

There was a couple new releases recently [to handle a newer Cython release](https://github.com/scikit-learn-contrib/hdbscan/releases).

So I was just testing if the new version was somehow breaking tests.